add quantization extra in the README (#300)
explosion/curated-transformers: README.md

## 🗜️ Quantization

`curated-transformers` supports dynamic 8-bit and 4-bit quantization of models by leveraging the [`bitsandbytes` library](https://github.com/TimDettmers/bitsandbytes).

> You can use the quantization variant to install the necessary dependencies for quantization with:
> 
> ```pip install curated-transformers[quantization]```

Reviewer: ```suggestion
Use the quantization variant to automatically install the necessary dependencies:

```bash
pip install curated-transformers[quantization]
```
```
